dbms/src/Common/TiFlashMetrics.h

```diff
@@ -232,7 +233,7 @@ namespace DB
         F(type_active_rpc_async_worker, {"type", "rpc_async_worker_active"}),                                                             \
         F(type_total_rpc_async_worker, {"type", "rpc_async_worker_total"}))                                                               \
     M(tiflash_task_scheduler, "Min-tso task scheduler", Gauge,                                                                            \
-        F(type_min_tso, {"type", "min_tso"}),                                                                                             \
+        F(type_min_query_ts, {"type", "min_query_ts"}),                                                                                   \
```

Comment:
> I'm not sure if it is a best practice to change the name of a existing metric instead of adding a new one.

Perhaps some users will get metrics from http API, I will keep it unchanged.